rosie.py

```diff
@@ -30,8 +29,11 @@ def run():
         help()
         exit(1)
     target_directory = argv[3] if len(argv) >= 4 else '/tmp/serenata-data/'
+    if '--years' in argv:
+        years = argv[argv.index('--years') + 1]
+        years = [int(num) for num in years.split(',')]
     klass = getattr(rosie, target_module)
-    klass.main(target_directory)
+    klass.main(target_directory, years=years)
```

Comment:
This is failing in tests:

```
$ python rosie.py run federal_senate
Traceback (most recent call last):
  File "rosie.py", line 62, in <module>
    command()
  File "rosie.py", line 36, in run
    klass.main(target_directory, years=years)
UnboundLocalError: local variable 'years' referenced before assignment
```

If no `--years` is passed, `years` variable is not set.